pkgs/tools/networking/dnsmasq/default.nix

```diff
@@ -32,6 +32,7 @@ stdenv.mkDerivation rec {
     "BINDIR=$(out)/bin"
     "MANDIR=$(out)/man"
     "LOCALEDIR=$(out)/share/locale"
+    "PKG_CONFIG=${buildPackages.pkgconfig}/bin/${buildPackages.pkgconfig.targetPrefix}pkg-config"
```

Comment:
Isn't there a `$PKG_CONFIG` variable available now? (So you don't have to have all that nix code.)

Reply:
i didnt find anything of the likes.

Reply:
Hmm. The setup hook from pkg-config will already define the environment variable. I guess from the look of it this Makefile might have trouble picking up definitions from the environment. Maybe try patching it to use `?=` instead of `=`?, and I bet we can get a lot of those `makeFlags`.

Reply:
good idea. just did and it seems to work. pushed changes

Reply:
Unless the patch goes upstream I actually prefer not patching the code over defining makeflags.

Reply:
see e.g. https://github.com/NixOS/nixpkgs/pull/91457 for the $PKG_CONFIG variable